src/types/inp.ts

```diff
@@ -37,6 +37,34 @@ export interface INPMetric extends Metric {
   entries: PerformanceEventTiming[];
 }
 
+export type INPSubpart =
```

Comment:
I still think we should name these `input-delay`, `processing-duration`, and `presentation-delay` to follow the [TAG casing rules](https://w3ctag.github.io/design-principles/#casing-rules).

I get that it would be inconsistent with the property names, but maybe that's less of an issue in this PR compared to the previous PR?

Reply:
Yeah I think it's less of an issue here. Done.